### PR TITLE
[Workflow] Update for JavaScript SDK

### PR DESCRIPTION
**Pull request by hhunter-ms**
## Description

- JS SDK presence in existing workflow docs
    - [x] How to author
    - [x] How to manage
    - [x] Workflow overview
    - [x] Workflow patterns
    - [ ] Quickstart
   
## Issue reference

PR will close: #3869

### CODE REVIEW

File: daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md [hunk 1]
```diff
@@ -52,6 +52,21 @@ def hello_act(ctx: WorkflowActivityContext, input):
...
+
+<!--javascript-->
+
+Define the workflow activities you'd like your workflow to perform. Activities are a function definition and can take inputs and outputs. The following example creates a counter (activity) called `hello_act` that notifies users of the current counter value. `hello_act` is a function derived from a class called `WorkflowActivityContext`.
+
+```javascript
+
+```
+
+[See the workflow activity in context.](https://github.com/dapr/python-sdk/blob/master/examples/demo_workflow/app.py#LL40C1-L43C59)
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md]: This is a Python reference. Did you mean to include javascript here?

File: daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md [hunk 2]
```diff
@@ -193,6 +208,21 @@ def hello_world_wf(ctx: DaprWorkflowContext, input):
 [See the `hello_world_wf` workflow in context.](https://github.com/dapr/python-sdk/blob/master/examples/demo_workflow/app.py#LL32C1-L38C51)
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md:224]: This is a Python reference. Did you mean to include javascript here?

File: daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md [hunk 3]
```diff
@@ -364,6 +394,25 @@ if __name__ == '__main__':
 ```
 
 
+{{% /codetab %}}
+
+{{% codetab %}}
+
+<!--javascript-->
+
+[In the following example](https://github.com/dapr/python-sdk/blob/master/examples/demo_workflow/app.py), for a basic JavaScript hello world application using the Go SDK, your project code would include:
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md]: Python again!

File: daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md [hunk 4]
```diff
@@ -364,6 +394,25 @@ if __name__ == '__main__':
...
+
+- A JavaScript package called `todo` to receive the Go SDK capabilities.
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md]: What is the "todo" here?

File: daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md [hunk 5]
```diff
@@ -504,5 +553,6 @@ Now that you've authored a workflow, learn how to manage it.
 - [Workflow API reference]({{< ref workflow_api.md >}})
 - Try out the full SDK examples:
   - [Python example](https://github.com/dapr/python-sdk/tree/master/examples/demo_workflow)
+  - [JavaScript example](todo)
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md]: What is the "todo" here?

File: daprdocs/content/en/developing-applications/building-blocks/workflow/howto-manage-workflow.md [hunk 6]
```diff
@@ -242,6 +260,7 @@ Learn more about these HTTP calls in the [workflow API reference guide]({{< ref
 - [Try out the Workflow quickstart]({{< ref workflow-quickstart.md >}})
 - Try out the full SDK examples:
   - [Python example](https://github.com/dapr/python-sdk/blob/master/examples/demo_workflow/app.py)
+  - [JavaScript example](todo)
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/workflow/howto-manage-workflow.md]: more todo here?

File: daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-overview.md [hunk 7]
```diff
@@ -93,18 +83,16 @@ Want to put workflows to the test? Walk through the following quickstart and tut
 | ------------------- | ----------- |
 | [Workflow quickstart]({{< ref workflow-quickstart.md >}}) | Run a workflow application with four workflow activities to see Dapr Workflow in action  |
 | [Workflow Python SDK example](https://github.com/dapr/python-sdk/tree/master/examples/demo_workflow) | Learn how to create a Dapr Workflow and invoke it using the Python `DaprClient` package. |
+| [Workflow JavaScript SDK example](todo) | Learn how to create a Dapr Workflow and invoke it using the JavaScript `todo` package. |
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-overview.md]: todo?

**Comment by hhunter-ms** [comment]
@kaibocai fyi

**Comment by hhunter-ms** [comment]
related PR: #3982

File: daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-overview.md [hunk 8]
```diff
@@ -73,16 +73,6 @@ Learn more about [different types of workflow patterns]({{< ref workflow-pattern
 
 The Dapr Workflow _authoring SDKs_ are language-specific SDKs that contain types and functions to implement workflow logic. The workflow logic lives in your application and is orchestrated by the Dapr Workflow engine running in the Dapr sidecar via a gRPC stream.
 
-### Supported SDKs
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-overview.md]: Are we going to put this back in?

File: daprdocs/content/en/getting-started/quickstarts/workflow-quickstart.md [hunk 9]
```diff
@@ -265,6 +265,113 @@ In `workflow.py`, the workflow is defined as a class with all of its associated
...
+
+```
+
+#### `order-processor/Activities` directory
+
+The `Activities` directory holds the four workflow activities used by the workflow, defined in the following files:
+
+## Watch the demo
+
+Watch [this video to walk through the Dapr Workflow .NET demo](https://youtu.be/BxiKpEmchgQ?t=2564):
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/quickstarts/workflow-quickstart.md]: Do you think we should remove this video given this is .NET?

**Review by kaibocai — APPROVED** [review]
LGTM thanks!

File: daprdocs/content/en/getting-started/quickstarts/workflow-quickstart.md (hunk 9 above)
Comment (hhunter-ms) [reply daprdocs/content/en/getting-started/quickstarts/workflow-quickstart.md]: I think it's helpful to have for .NET - I can move into the .NET tab so it only shows up when that SDK is selected

**Review by msfussell — APPROVED** [review]
Approved!

**Pull request merged**